src/style.css

```diff
@@ -0,0 +1,95 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+}
+
+.search-form {
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background: #fff;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+
+  display: flex;
+  align-items: center;
+  text-align: center;
+
+  height: 70px;
+  margin: 20px 10px;
+}
+
+.search-form--small {
+  height: 42px;
+}
+
+.input {
+  background: transparent;
+  flex: 1;
+  border: 0;
```

Comment:
do you need it? I mean, does it have a default border?

Reply:
Without border: 0; I have a border in input